packages/core/admin/server/middlewares/rateLimit.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const utils = require('@strapi/utils');
+const { toLower } = require('lodash/fp');
+
+const { RateLimitError } = utils.errors;
+
+module.exports =
+  (config, { strapi }) =>
+  async (ctx, next) => {
+    const ratelimit = require('koa2-ratelimit').RateLimit;
+
+    const userEmail = toLower(ctx.request.body.email) || 'unknownEmail';
+
+    return ratelimit.middleware({
+      interval: { min: 5 },
+      max: 5,
```

Comment:
Could you add a comment to indicate the unit ? is it seconds or milliseconds ? 🤔

Reply:
It's maximum 5 requests per 5 minutes :)
The time unit is in line 16 `min = minutes` ([doc](https://github.com/ysocorp/koa2-ratelimit#time-type))